### PR TITLE
3rdparty/symfony/translation

### PR DESCRIPTION
**Pull request by vitormattos**
# Bump [symfony/translation](https://github.com/symfony/translation)

Ref: https://github.com/nextcloud/3rdparty/pull/1304
Close https://github.com/nextcloud/3rdparty/pull/922

The original PR from dependabot used an old version of translation package. I created a new PR with the newest version of package compatible with PHP 8.0. 

The class `TranslatorInterface` that I removed from comment was removed from Translation package.

### CODE REVIEW

**Comment by nickvergessen** [comment]
Feature freeze, try again when stable 26 is branched off end of the month 😅

**Comment by come-nc** [comment]
Please rebase @vitormattos

**Comment by vitormattos** [comment]
@come-nc rebased.

File: lib/private/L10N/L10NString.php [hunk 1]
```diff
@@ -57,7 +57,6 @@ public function __toString(): string {
 		$translations = $this->l10n->getTranslations();
 		$identityTranslator = $this->l10n->getIdentityTranslator();
 
-		// Use the indexed version as per \Symfony\Contracts\Translation\TranslatorInterface
```
Comment (nickvergessen) [inline lib/private/L10N/L10NString.php:60]: Can the comment be adjusted to the new source instead?

**Review by come-nc — APPROVED** [review]

**Pull request closed without merging**